test/jdk/java/lang/management/ThreadMXBean/VirtualThreadDeadlocks.java

```diff
@@ -40,6 +40,7 @@ public class VirtualThreadDeadlocks {
     private static final Object LOCK1 = new Object();
     private static final Object LOCK2 = new Object();
 
+    private static volatile boolean lock2Obtained = false;
```

Comment:
The term used in the APIs is "lock held"  so might be better to rename this to lock2Held. Can you drop the explicit init to false and also put a blank line after the declaration as it looks strange to have it run into the comment on the main method.